### ISSUE TITLE
add "shoot yourself in the foot mode" (send arbitrary json)

### ISSUE THREAD

**Issue by yargevad** [SparkPost/gosparkpost#68]
This will allow use of every API feature, specifically ones that don't have explicit support in the client library yet.

**Comment by kakysha** [comment]
Isn't it already implemented in standard library ?)

**Comment by yargevad** [comment]
Yes, `net/http` would let you do this. The goal here is that if current users of `gosparkpost` would like to use new features that the library doesn't yet support, then they don't have to switch everything over to `net/http`
